expr/expr.go

```diff
@@ -3266,6 +3267,74 @@ func EvalExpr(e *expr, from, until int32, values map[MetricRequest][]*MetricData
 		}
 
 		return results, nil
+
+
+	case "cactiStyle": // cactiStyle(seriesList, system=None, units=None)
+		// Get the series data
+		original, err := getSeriesArg(e.args[0], from, until, values)
+		if err != nil {
+			return nil, err
+		}
+
+		// Get the arguments
+		var system string = ""
+		var unit string = ""
+		if len(e.args) >= 2 {
+			system = e.args[1].valStr
+		}
+		if len(e.args) >= 3 {
+			unit = e.args[2].valStr
+		}
+
```

Comment:
Please make use of the existing `getStringNamedOrPosArgDefault()` function to set these variables. Ex:
```go
        system, err := getStringNamedOrPosArgDefault(e, "system", 1, "")
        if err != nil {
            return nil, err
        }
```

They're mouthfulls, but will allow calls using named params:
* cactiStyle( seriesList, "si", units="firkins")
* cactiStyle( seriesList, units="furlongs")

// With the known limitation that cactiStyle( seriesList, system="si", "fortnights" ) will fail.